File: QRCodeApp.API/Models/PhoneLineType.cs
namespace QRCodeApp.API.Models
{
    public static class PhoneLineType
    {
        public const string Mobile = "Mobile";
        public const string Landline = "Landline";

        public static bool IsValid(string? v) =>
            string.Equals(v, Mobile, StringComparison.OrdinalIgnoreCase) ||
            string.Equals(v, Landline, StringComparison.OrdinalIgnoreCase);

        public static string Normalize(string? v) =>
            string.Equals(v, Landline, StringComparison.OrdinalIgnoreCase) ? Landline : Mobile;
    }
}

File: publish/hostinger-bundle/api/src/PhoneLineType.php
<?php

declare(strict_types=1);

namespace QrApp;

final class PhoneLineType
{
    public const MOBILE = 'Mobile';
    public const LANDLINE = 'Landline';

    public static function isValid(?string $v): bool
    {
        if ($v === null) {
            return false;
        }

        return strcasecmp($v, self::MOBILE) === 0 || strcasecmp($v, self::LANDLINE) === 0;
    }

    public static function normalize(?string $v): string
    {
        return strcasecmp((string) $v, self::LANDLINE) === 0 ? self::LANDLINE : self::MOBILE;
    }
}
